src/menu/chapter-thumbnail-menu-item.js

```diff
@@ -28,15 +28,32 @@ class ChapterThumbnailMenuItem extends VjsMenuItem {
     super(player, {
       ...options,
 
-      label: chapterThumbnailTemplate(cue, options),
       selected: (cue.startTime <= currentTime && currentTime < cue.endTime),
+      template: chapterThumbnailTemplate(cue, options),
     });
 
-    this.addClass('vjs-chapter-thumbnails-menu-item');
-
     textTrack.addEventListener('cuechange', videojs.bind(this, this.onCueChange));
   }
 
+  createEl(type, props, attrs) {
+    const { template } = this.options_;
+
+    const el = super.createEl('li', Object.assign({
+      className: 'vjs-menu-item vjs-chapter-thumbnails-menu-item',
+      innerHTML: '', // does this need to be localized?
```

Comment:
This use to get passed into as a `label` and then was passed through `this.localize` as seen in https://github.com/videojs/video.js/blob/4f6cb03adde9ddf800e2ecf6fa87b07d436b74e8/src/js/menu/menu-item.js#L45.

Since `innerHTML` only accepted `string`'s. This was refactored to leverage `insertBefore` (losing this ability).

#breadcrumb